### DISCUSSION TITLE
Install docker mailserver with dokku paas?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [docker-mailserver/docker-mailserver#3187] [Q&A]
For those who do not know it dokku.com is a gem, opensource paas / heroku clone.

Could i install docker mailserver using a dockerfile installation method on dokku? Its badically a docker container management tool so it should be possible right? If interested i can document the steps and try this.

was just curious if i run into any problems or things i have to take into considerstion before attempting it.

my plan is to run dokku mailserver eith as minimal memory isage disabling anti virus then use it on my dokku vps to manage all of my mailboxes in 1 central place.

**Comment by polarathene** [comment]
- We support `compose.yaml` + `docker run` methods officially with Docker as the container runtime.
- Some users use Podman instead, and encounter compatibility issues. Others have used Kubernetes and need to rely on some workarounds like `OVERRIDE_HOSTNAME`.
- Maintainers do not have the bandwidth to support anything they do not use themselves. You are welcome to contribute alternative deployment methods to our docs though!

Your best bet is to try get it working with Dokku if you're comfortable doing that. My understanding with Dokku was that it simplifies deploying a container, which is useful to a broader audience often not too familiar with containers / devops, but wanting ease of deployment, especially if it's an image that can be leveraged for multiple instances (eg: NodeJS).

DMS seems less suitable for that demographic, unless you are running multiple mail containers and able to configure them appropriately (_on the same system this may get complicated with port management, and users find issues with container IPs only being visible to services like Fail2Ban unless they avoid proxies or use host mode networking, which would not be compatible with multiple instances_).

If this is just for you, chances are it's not worth the effort to pursue.

**Reply by Jarrodsz** [reply]
I just want to run a single docker mailserver container, to provide for all my now and future mail needs. 1 central place and my mails on my own server instead of some ISP. I try to get it up and running with dokku and will report back anything I find along the way.
